lib/components/util/formatted-mode.tsx

```diff
@@ -0,0 +1,18 @@
+import { useIntl } from 'react-intl'
+import React, { ReactElement } from 'react'
+
+import { getFormattedMode } from '../../util/i18n'
+
+interface Props {
+  mode: string
+}
+
+/**
+ * JSX wrapper for the i18n.getFormattedMode function (renders into a React fragment).
+ */
+const FormattedMode = ({ mode = '' }: Props): ReactElement => {
+  const intl = useIntl()
+  return <>{getFormattedMode(mode, intl)}</>
+}
+
+export default FormattedMode
```

Comment:
What if we moved this into the file where getFormattedMode() is? Or are we keeping the components separate?

Reply:
Or should we just get rid of this component?

Reply:
No, that would add extra lines for adding `useIntl()`. It's better to keep components separate imo.